core/datainput.gms

```diff
@@ -290,16 +290,24 @@ $offdelim
 loop(te$(fm_dataglob("constrTme",te) > 0),
   p_tkpremused(regi,te) = 1/fm_dataglob("constrTme",te)
     * sum(integ$(integ.val <= fm_dataglob("constrTme",te)),
-$if %cm_techcosts% == "REG"  (1 + 0.02/pm_ies(regi) + pm_prtp(regi) + p_risk_premium_constr(regi) )  ** (integ.val - 0.5) - 1
-$if %cm_techcosts% == "GLO"  (1 + 0.02/pm_ies(regi) +  pm_prtp(regi) )                               ** (integ.val - 0.5) - 1
+$ifthen %cm_techcosts% == "GLO"
+    (1 + 0.02/pm_ies(regi) +  pm_prtp(regi) )                               ** (integ.val - 0.5) - 1
+$else
+    (1 + 0.02/pm_ies(regi) + pm_prtp(regi) + p_risk_premium_constr(regi) )  ** (integ.val - 0.5) - 1  
+$endif
       )
```

Comment:
I would leave the older formulation, since it could be that we have a third switch option in the future (besides GLO and REG), this way is might be clearer

Reply:
yes, do you know how to write "if it's REG or REG2040"? I understood [here](https://forum.gamsworld.org/viewtopic.php?t=11859) that the "or" is a bit messy with these pre-compiler instructions...

Reply:
https://github.com/remindmodel/remind/blob/master/modules/80_optimization/nash/solve.gms#L13

i only found this example (on master branch :sweat_smile: ) not sure if it works here..

Reply:
probably just try it?